Trading/Exchange/bybit.py

```diff
@@ -46,6 +50,8 @@ class Bybit(FutureExchange):
     BYBIT_FUNDING_TIMESTAMP = "funding_rate_timestamp"
     BYBIT_DEFAULT_FUNDING_TIME = 8 * HOURS_TO_SECONDS
 
+    BYBIT_TIME = "time_now"
```

Comment:
😮

Reply:
https://bybit-exchange.github.io/docs/inverse/#t-latestsymbolinfo

Reply:
Yep, it's just a surprising naming :)